app/services/analytics_events.rb

```diff
@@ -1537,6 +1541,7 @@ def idv_front_image_clicked(
     isDrop:,
     source:,
     use_alternate_sdk:,
+    captureAttempts:,
```

Comment:
nit: as idv_front_image_clicked and idv_back_image_clicked they have the same args, can we keep the argument list ordering the same